app/views/spree/admin/gdpr_requests/index.html.erb

```diff
@@ -14,15 +14,17 @@
   <table class="index" id="listing_gdpr_requests">
     <colgroup>
        <col style="width: 15%;">
+       <col style="width: 5%;">
        <col style="width: 15%;">
        <col style="width: 20%;">
        <col style="width: 20%;">
-       <col style="width: 20%;">
+       <col style="width: 15%;">
        <col style="width: 10%;">
     </colgroup>
     <thead>
       <tr data-hook="admin_products_index_headers">
-        <th><%= Spree::GdprRequest.human_attribute_name(:user) %></th>
+        <th><%= Spree::GdprRequest.human_attribute_name(:email) %></th>
+        <th><%= I18n.t('spree.listing_orders') %></th>
```

Comment:
I don't know if showing the number of orders here makes much sense. I was under the impression you'd only show it when the admin is entering an email, to help them choose the right one.